### ISSUE TITLE
Make the POC ConfigsManager as the new ConfigsManager Step 1

### ISSUE THREAD

**Issue by dillu24** [SimplyVC/panic#147]
### Technical Story

The alerter needs to be notified whenever there is a configuration change.

### Description

Since we are now storing the configurations inside the database we must be able to detect a configuration change by listening to database events. This means that we need to migrate the POC logic that @simplyrider has developed and replace the old `ConfigsManager`

### Requirements

- [ ] The new configs manager is able to map the structure of the new configurations into the old structure. This is needed so that the alerter can continue operating without any changes.
- [ ] The new configs manager is able to detect a change inside a configuration and send the entire configuration to the `CONFIG_EXCHANGE` with the correct routing key
- [ ] Connect the new `ConfigsManager` with `run_alerter.py` in order for the manager to start

**Extra attention should be put towards the following**:

- On startup we need to send all configurations from the database.
- Error handling should be implemented similarly to the old `ConfigsManager`
- Proper thread handling of the newly created threads.
- Dynamic RabbitMQ routing keys based on configs.

### Acceptance criteria

**Given**:  Any Substrate configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any Cosmos configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any Chainlink configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Given**:  Any General configuration change
**Then**: The `ConfigsManager` sends the correct configuration update with the correct routing key

**Comment by dillu24** [comment]
Part of this ticket is to create a presentation to the team so that everyone is aware of what needs to be done.

**Comment by itsciccio** [comment]
Not merged. Will first have https://github.com/SimplyVC/panic/issues/334 merged into it since as-is, heartbeats do not work. Once this is merge is complete, we are to merge this branch into `ui_iteration_2`.
